### PR TITLE
ci: Sync schema changes to code-intel-extensions.

### PR DESCRIPTION
**Pull request by varungandhi-src**
Based on Robert's suggestion in sourcegraph/code-intel-extensions#791.

## Test plan

Test in ~prod~ CI.

### CODE REVIEW

**Review by bobheadxi — APPROVED** [review]
nice, that was fast!

**Comment by varungandhi-src** [comment]
Merging this now; we can tweak it later if @davejrt has any suggestions for improvement.

**Comment by davejrt** [comment]
retrospective thumbs up, sorry for the delay!

File: .github/workflows/push-schema.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: 'sourcegraph/code-intel-extensions'
+        env:
+          GITHUB_TOKEN: ${{ secrets.BOT_GITHUB_TOKEN }}
+      - name: Create PR if applicable
+        run: |
+          wget -o schema/schema.graphql https://github.com/sourcegraph/sourcegraph/blob/main/cmd/frontend/graphqlbackend/schema.graphql
```
Comment (efritz) [inline .github/workflows/push-schema.yml:25]: See https://github.com/sourcegraph/code-intel-extensions/blob/master/package.json#L25

**Pull request merged**